specification/dartLangSpec.tex

```diff
@@ -14081,6 +14084,23 @@ \subsection{Property Extraction}
 (\ref{superGetterAccessAndMethodClosurization}).
 \EndCase
 
+\LMHash{}%
+\Case{Implicit}
+Let $e$ be an expression whose static type is
+an interface type that has a method named \CALL.
+In the case where the context type for $e$ is a function type,
+$e$ is treated as \code{$e$.\CALL}.
+
+\commentary{%
+This means that a ``callable object'' may be treated as a function
+that supports a mechanism similar to function closurization
+(\ref{functionClosurization})
+by desugaring it to a method closurization on \CALL.
+It is required that it is statically known that it is a callable object,
```

Comment:
&mapsto; It must be statically known that the value is ...

To many "it"s.

Reply:
Done!